mozilla/fx-private-relay-add-on
# [MPP-1708] Popup Refactor - Login Panel

## Summary

Links:
- Jira Tickets:
  - https://mozilla-hub.atlassian.net/browse/MPP-2819
- L10N PR: https://github.com/mozilla-l10n/fx-private-relay-add-on-l10n/pull/44
- Epic PR #456 

## Testing

- Run add-on
- DO NOT Log in from first-run page
- Open popup
- **Expected:** Panel should display styled login prompt, and have two nav bar items (Settings, Open Relay)
- Click "Log in" button
- **Expected:** New tab should open with to FxA login flow
- Once logged in, open panel one more time
- **Expected:** Should display masks list (or if no masks are created, an empty state to generate a random mask)

## Screenshots

Log-in Screen: 

<img width="408" alt="image" src="https://user-images.githubusercontent.com/2692333/216707256-3d2be67c-2310-4541-ac39-f889a5470cba.png">

Additional fixes: 

- Secondary button: Removed background color

<img width="406" alt="image" src="https://user-images.githubusercontent.com/2692333/216707415-8e50d604-1259-45f9-8de2-11838939ac8d.png">

- News unread notification bubble color

<img width="77" alt="image" src="https://user-images.githubusercontent.com/2692333/216707561-fad94db9-470e-4611-8226-819eef371d9b.png">
